test/fixtures/rules/css/test.css

```diff
@@ -0,0 +1,7 @@
+.usedClass{
+    color:green
```

Comment:
Add a space after `:` and `;` at the end of the line. (here and below)

Reply:
fixed